src/components/SignUpSuccessForm/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const SignUpSuccessForm = () => {
+  return (
+    <section className="sign-up-success-form">
+      <header className="sign-up-success-form__header">Sample Picture</header>
+      <div className="sign-up-success-form__body">
+        <h2>Registration Success</h2>
+      </div>
+      <footer className="sign-up-success-form__footer">
+        <div className="form__actionable">
+          <button className="form__button">Continue</button>
+        </div>
+      </footer>
```

Comment:
Since there's no `form` so I don't think we need to this. How about we make this button a `sign-up-success-form` element?
```suggestion
      <div className="sign-up-success-form__actionable">
        <button className="sign-up-success__button">Continue</button>
      </div>
```